papaparse.js

```diff
@@ -57,6 +57,7 @@
 	Papa.BAD_DELIMITERS = ['\r', '\n', '"', Papa.BYTE_ORDER_MARK];
 	Papa.WORKERS_SUPPORTED = !IS_WORKER && !!global.Worker;
 	Papa.SCRIPT_PATH = null;	// Must be set by your code if you use workers and this lib is loaded asynchronously
+	Papa.SCRIPT_PATH_AS_BLOB_URL = false;	// Should be used when you want to use blob urls as 'SCRIPT_PATH' location.
```

Comment:
Can we remove the SCRIPT_PATH_AS_BLOB_URL please?

Reply:
Man, I don't know. If we remove this, it will always load as a blob url?

Honestly, I kind of lost the interest in making this PR, because it seems like I was the only one having this problem, it was very specific to my situation.

But if you still think it is still useful, I can try removing that.

Reply:
Yes, we should load always as blob url. 

Do you think there is any drawback of doing so?